app/src/main/java/de/westnordost/streetcomplete/data/OsmModule.java

```diff
@@ -62,18 +65,18 @@ public static OsmConnection osmConnection(OAuthConsumer consumer)
 	}
 
 	@Provides public static OverpassMapDataDao overpassMapDataDao(
-			Provider<OverpassMapDataParser> parserProvider)
+			Provider<OverpassMapDataParser> parserProvider, SharedPreferences prefs)
 	{
 		OsmConnection overpassConnection = new OsmConnection(
-				OVERPASS_API_URL, ApplicationConstants.USER_AGENT, null);
+			prefs.getString(Prefs.OVERPASS_URL, "https://overpass-api.de/api/"), ApplicationConstants.USER_AGENT, null);
```

Comment:
```suggestion
			prefs.getString(Prefs.OVERPASS_URL, OVERPASS_API_URL), ApplicationConstants.USER_AGENT, null);
```
?

Reply:
```suggestion
			prefs.getString(Prefs.OVERPASS_URL, OVERPASS_API_URL), ApplicationConstants.USER_AGENT, null);
```

direct suggestion :wink: